bin/dev_build

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# *****NOTE:
+# This is functionally almost identical to the devkit binary. and should be deprecated
+# in the near future by just updating devkit
+#
+
+load File.expand_path(File.join(File.dirname(__FILE__), "..", "lib", "vanagon.rb"))
```

Comment:
This is not a reliable way to require a library. There's no guarantee that this path will actually resolve. What's wrong with `require 'vanagon'`?